modules/psap-special-resource-operator-using.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="using-the-special-resource-operator_{context}"]
+= Using Special Resource Operator
```

Comment:
Per slack thread, suggest looking through and adding "the" in front of SRO (or spelled out) for consistency.